src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -1071,4 +1076,44 @@ def __update_patches_and_errors_in_substatus(self, substatus_msg, substatus_msg_
     def __get_errors_from_substatus(self, substatus_msg):
         """ Get errors code and errors details from substatus message json """
         return substatus_msg['errors']['code'], substatus_msg['errors']['details']
+
+    def __create_assessment_tombstones_by_classification(self, packages_removed_from_assessment):
+        """ Create list of tombstone per classification with max count of that classification, omit unclassified """
+        assessment_tombstone_map = {}
```

Comment:
This is actually storing a map between classifications and the count of patches for it. The name 'assessment_tombstone_map' indicates it may have some tombstone message, which is not the case here.

Rename this to assessment_classification_count_map or classification_count_map

Reply:
Don't see this change reflected yet @feng-j678

Reply:
I actually refactor method name instead of the variable, now it is updated and i also change it to ordermap to ensure order of tombstone [critical, security, other] because py27 don't maintain order

done